test_scripts/Resumption/Handling_errors_from_HMI/044_Same_RPC_IVD_RAI2_before_request_to_HMI.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description:
+-- In case:
+-- 1. App1 and App2 subscribed to <RPC>
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App1 and App2 re-register with actual HashId
+-- 4. Resumption for App1 and App2 is started:
+--    <RPC> related to App1 is sent from SDL to HMI
+-- 5. HMI responds with error resultCode
+-- 6. SDL doesn't send revert <RPC> request to HMI
+-- 7. SDL doesn't restore subscription to <RPC> and responds RAI_Response(success=true,resultCode=RESUME_FAILED) to App1
+-- 8. SDL continue resumption for App2:
+--    <RPC> related to App2 is sent from SDL to HMI
```

Comment:
@GetmanetsIrina Why <RPC> related to App2 is sent after SDL received response of the <RPC> related to App1 here? It is not according the test script.

Reply:
it's according to scripts.
There is the next sequence in log for this script:
[15:09:54,531] RAI 1
[15:09:54,542] RAI 2
[15:09:54,563] BC.OnAppRegistered 1
[15:09:55,573] RC.GetInteriorVehicleData
[15:09:55,591] RC.GetInteriorVehicleData: GENERIC_ERROR
[15:09:55,711] RAI 1: RESUME_FAILED
[15:09:55,730] BC.OnAppRegistered 2
[15:09:56,611] RC.GetInteriorVehicleData
[15:09:56,611] RC.GetInteriorVehicleData: SUCCESS
[15:09:56,890] RAI 2: SUCCESS